index.html

```diff
@@ -131,6 +131,32 @@ <h2 class="connect">LET'S CONNECT</h2>
             </ul>
             </div>
         </section>
+        <Section class="form-Container">
+            <form id="form" action="https://formspree.io/f/mwkyoojy" method="post">
+                <h2 id="contact-Me">Contact Me</h2>
+                <p class="contact-Me-Paragraph">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+                <ul class="form-Ul">
+                    <li class="form-Li">
+                      <label for="name"></label>
+                      <input type="text" id="name" name="user_name" placeholder="Full Name" required>
+                    </li>
+                    <li class="form-Li">
+                      <label for="mail"></label>
+                      <input type="email" id="mail" name="user_email" placeholder="example@gmail.com" required>
+                    </li>
+                    <li class="form-Li">
+                      <label for="msg"></label>
+                      <textarea id="msg" name="user_message" placeholder="Write your message here..." required></textarea>
```

Comment:
- [ ] As it is a requirement, please add a `maxlength` attribute to the `textarea` with the right value

Reply:
Added